### DISCUSSION TITLE
Select component options in dark mode

### DISCUSSION THREAD

**Discussion by LechK** [twilio-labs/paste#2909] [Ideas] [answered]
![select in library](https://user-images.githubusercontent.com/46899047/210217728-a375bbb5-8580-4a7d-a031-c21191534d41.png)
![Screenshot 2023-01-02 120303](https://user-images.githubusercontent.com/46899047/210217735-f6d69b18-6914-4014-996b-3c838257513c.jpg)
Hello team!
Select component options in dark mode are displayed really not user friendly.
Is it supposed to work like that, or I'm missing something?
I have attached two files, from your documentation and twilio project

**Comment by TheSisb** [comment]
Hi @LechK 

I wasn't able to reproduce this on either Chrome or Firefox. e.g.:
<img width="978" alt="image" src="https://user-images.githubusercontent.com/1592327/210474378-97703dd3-e08d-4b0c-9df3-20c144781b3f.png">

Are you using the dark theme from our page header or a dark mode browser extension/setting? Also which browser was this on?

Thanks for the report!

**Reply by LechK** [reply]
Hi @TheSisb ! Thank You for your answer!
on Paste page, I was switching theme by toggle button at top of page. That was Chrome.
On Twilio Flex, after 2.0.2 upgrade (when Material select was replaced by Paste select) same issue appeared. It was reported by QA, so it's not only me, who has that issue

**Comment by justinduncan2010** [comment, marked as answer]
I am having the same issue, except it is with the production build only and only from version 2.0.0-beta.2 forward. The develop build and previous versions use a material UI component, which looks fine in dark mode.
**Production:**
![image](https://user-images.githubusercontent.com/12818159/214580827-17791a37-1585-417d-b44e-4b88cefa4eea.png)

**Develop:**
![image](https://user-images.githubusercontent.com/12818159/214581095-fa6c8784-5045-40f2-88f9-dfceee6d6453.png)

**Reply by gloriliale** [reply]
Hi @justinduncan2010!

Paste's Select component uses the native html picker, so the options list is styled according to the browser default (in contrast, material ui does not use the browser default). Could you tell me a little more about your implementation of dark mode, and the browser you're using for these examples?

**Reply by LechK** [reply]
I have checked on all browsers on my PC (Edge, Chrome, Mozilla), same issue on all of them

**Reply by LechK** [reply]
Or could it be because of Windows OS?

**Reply by gloriliale** [reply]
Hi @LechK! Thanks for letting us know the browsers you're using where this shows up. Are you saying that you're having this issue in Windows OS but not on MacOS? 
Additionally, are you using any other browser extension/setting to set dark mode on the site?

**Reply by LechK** [reply]
Sorry @gloriliale , I should have added more info. We don't have MacOS'es to test that, so that's why I have asked about that, is it possible that it is related to Windows/Mac OS's. 
About browser extensions/settings, I have tried with dark/light/default settings, with no extensions at all - same result.

**Reply by kelseycavitt** [reply]
Hi @LechK!
We've added a ticket to our backlog to address this, there's a bug with the Select list in dark mode on Windows. Someone on our team will follow up once that's been released. Thanks for finding this and for your patience!

**Reply by stjohnson44** [reply]
Hi @kelseycavitt. Following up on this to see if there is any estimated timeline for the fix, or if there are any workarounds in the meantime.

**Reply by SiTaggart** [reply]
Hi there

This is fixed in Paste and released recently. If you are still seeing it in products like Flex, it's because that dependency needs to be updated to the latest Paste version.

That would be a question for the Flex product team at that point.

Thanks

**Reply by stjohnson44** [reply]
Hi @SiTaggart 

I updated to Paste version 19 and am still seeing this issue. The component is built entirely with Paste elements, but is being used inside of a Flex instance. The component itself is not using any parts of the flex-ui library. Would that still be an issue for the Flex product team?

**Reply by SiTaggart** [reply]
Hi

I totally messed up and got our releases confused. _Now_, it's released in core v19.0.1

Hopefully that solves your issue.

Thanks

**Reply by stjohnson44** [reply]
No worries Simon. Updating to 19.0.1 did indeed fix the issue. Thank  you!
